### PR TITLE
Fix https://wso2.org/jira/browse/ML-101

### PR DESCRIPTION
**Pull request by cdathuraliya**

### CODE REVIEW

File: components/ml/org.wso2.carbon.ml.core/src/main/java/org/wso2/carbon/ml/core/spark/algorithms/SparkModelUtils.java [hunk 1]
```diff
@@ -421,4 +421,19 @@ public Boolean call(Tuple2<Double, Double> pl) {
...
+     * @param value     Value of the key
+     */
+    public static <T, E> T getKeyByValue(Map<T, E> map, E value) {
+        for (Map.Entry<T, E> entry : map.entrySet()) {
+            if (Objects.equals(value, entry.getValue())) {
+                return entry.getKey();
+            }
+        }
+        return null;
+    }
```
Comment (nirmal070125) [inline components/ml/org.wso2.carbon.ml.core/src/main/java/org/wso2/carbon/ml/core/spark/algorithms/SparkModelUtils.java]: @cdathuraliya can we move this to org.wso2.carbon.ml.core.utils.MLUtils since this is not related to Spark?

Comment (cdathuraliya) [inline components/ml/org.wso2.carbon.ml.core/src/main/java/org/wso2/carbon/ml/core/spark/algorithms/SparkModelUtils.java]: @nirmal070125 Moved. :)
